frontend/src/components/pomodoroview/PomodoroView.jsx

```diff
@@ -89,26 +94,36 @@ function RemainingTimeContextProvider(props) {
 }
 
 function ClockController() {
-  const { startTime, setStartTime, setClockKey, setIsClockPlaying, isUserBreaking, setIsUserBreaking } = useContext(
-    remainingTimeContext
-  );
+  const {
+    startTime,
+    setStartTime,
+    setClockKey,
+    setIsClockPlaying,
+    isUserBreaking,
+    setIsUserBreaking,
+    isClockPlaying
+  } = useContext(remainingTimeContext);
 
-  const handleGo = () => {
-    setIsClockPlaying(true);
-    setStartTime(getCurrentTime());
-  };
-  const handleHold = () => {
-    if (!isUserBreaking) {
-      submitRecord(startTime, true);
+  const [isGoing,setChangeGo] = useState('Go') // change Go/Hold button
+  const handleGo = (flag) => {
```

Comment:
change function name from handleGo to HandleSwitch, because it is actually a switch.